src/insert/comments-expanded-advert.ts

```diff
@@ -121,13 +121,8 @@ const removeMobileCommentsExpandedAds = (): Promise<void> => {
 			log('commercial', `Removing ad slot: ${node.id}`);
 			const advert = getAdvertById(node.id);
 			if (advert) {
-				window.googletag.destroySlots([advert.slot]);
+				emptyAdvert(advert);
 			}
-			node.remove();
-			dfpEnv.adverts.delete(node.id);
-			dfpEnv.advertsToLoad = dfpEnv.advertsToLoad.filter(
-				(_) => _ !== advert,
-			);
 		}),
```

Comment:
Nice, it looks tidier now we've removed the `dfpEnv` logic from here as we use it via `emptyAdvert` now 👍